presto-native-execution/presto_cpp/main/types/tests/RowExpressionTest.cpp

```diff
@@ -18,13 +18,17 @@
 #include "presto_cpp/presto_protocol/core/presto_protocol_core.h"
 #include "velox/core/Expressions.h"
 #include "velox/type/Type.h"
+#include "velox/functions/prestosql/types/JsonType.h"
 
 using namespace facebook::presto;
 using namespace facebook::velox;
 using namespace facebook::velox::core;
 
 class RowExpressionTest : public ::testing::Test {
  public:
+  RowExpressionTest(){
+    registerJsonType();
+  }
   static void SetUpTestCase() {
     memory::MemoryManager::testingSetInstance({});
```

Comment:
May be we can register in setup and remove the constructor.

Reply:
Updated this